package.json

```diff
@@ -124,12 +124,13 @@
     "ng2-bootstrap": "1.3.3",
     "ng2-restangular": "0.1.30",
     "ngx-base": "1.2.9",
-    "ngx-dropdown": "0.0.22",
+    "ngx-bootstrap": "1.7.1",
```

Comment:
Do you realize that you swapped out "dropdown" for "bootstrap" and left the old version of "bootstrap" in place?